doc/onboarding.md

```diff
@@ -223,9 +222,9 @@ onboarding session.
   * [https://github.com/nodejs/LTS](https://github.com/nodejs/LTS)
   * [https://github.com/nodejs/citgm](https://github.com/nodejs/citgm)
 * The Node.js Foundation hosts regular summits for active contributors to the
-  Node.js project, where we have face-to-face discussion about our work on the
-  project. The foundation has travel funds to cover participants' expenses
-  including accommodation, transportation, visa fees etc. if needed. Check out
+  Node.js project, where we have face-to-face discussions about our work on the
+  project. The Foundation has travel funds to cover participants' expenses
+  including accommodations, transportation, visa fees, etc. if needed. Check out
```

Comment:
Is `accommodations` US English? Genuine question -- it looks odd to me but I am British.

Reply:
@richardlau I did not realize that was a peculiarity of US English, but yes, apparently it is. I've never seen it without the `s` at the end. Checked online in a few sources and it is indeed a US English tendency to add the `s` no matter what while the rest of the world doesn't do that.  

Perhaps we can change it to `lodging` and it won't seem odd to anyone? :-D

Reply:
I'm fine with keeping it as the docs should be written in American English. Only now I'm staring at `transportation` and wondering if that should end in `s` 😆.